src/main/java/seedu/main/Parser.java

```diff
@@ -61,6 +64,43 @@ public Command parseCommand(String commandPart) {
         return commands.get(commandPart);
     }
 
+
+    public Command parseCommand(String commandPart, String argumentString, TransactionEditor transactionEditor) {
```

Comment:
We already have a parser that works for every commands. You may try to follow the format of other command classes instead of adding a new way to parse